module/VuFind/src/VuFind/Db/Entity/Comments.php

```diff
@@ -38,50 +38,72 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="comments", indexes={@ORM\Index(name="resource_id", columns={"resource_id"}), @ORM\Index(name="user_id", columns={"user_id"})})
+ * @ORM\Table(name="comments",
+ * indexes={@ORM\Index(name="resource_id", columns={"resource_id"}),
+ * @ORM\Index(name="user_id",  columns={"user_id"})}
+ * )
  * @ORM\Entity
  */
 class Comments implements EntityInterface
 {
     /**
+     * Unique ID.
+     *
      * @var int
      *
-     * @ORM\Column(name="id", type="integer", nullable=false)
+     * @ORM\Column(name="id",
```

Comment:
By contrast to earlier entities, you've kept the name here. We should probably be consistent one way or the other.